lib/ex_doc/cli.ex

```diff
@@ -109,6 +109,8 @@ defmodule ExDoc.CLI do
       -m, --main          The main, entry-point module in docs,
                           default: "extra-api-reference" when --formatter is "html"
       -p, --homepage-url  URL to link to for the site name
+          --extra-title   Allow user to define the title for the additional Markdown files,
```

Comment:
Is `--extra-title` the best name for it? Maybe `--extra-section`? Also, don't forget to change the docs for the mix task too.